capstone/src/components/User/Adventurer/FindAdventure/FindAdventure.jsx

```diff
@@ -0,0 +1,34 @@
+import './FindAdventure.css';
+import { useState } from 'react';
+import Map from './Map/Map';
+
+export default function FindAdventure() {
+  const [displayExperience, setDisplayExperience] = useState('map');
+  return (
+    <div className="findAdventure">
+      <h2>Find your next adventure:</h2>
+      <ExperienceDisplayer
+        displayExperience={displayExperience}
+        setDisplayExperience={setDisplayExperience}
+      />
+      <Map />
+    </div>
+  );
+}
+
+export function ExperienceDisplayer({ displayExperience, setDisplayExperience }) {
+  if (displayExperience === 'map') {
+    return (
+      <div className="experienceDisplayer">
+        <button type="button" value="map" className="chosenDisplay" disabled>Map</button>
+        <button type="button" value="list" className="notChosenDisplay" onClick={(e) => setDisplayExperience(e.target.value)}>List</button>
```

Comment:
Let's avoid reading from the event object, it should be unnecessary most of the time. We know here that the value should be "list".

It also looks like we're repeating a lot of unnecessary markup and code. You can use [composition](https://reactjs.org/docs/composition-vs-inheritance.html) here to build up your components.

```
function ExperienceViewButton(props /* { value: string, children: string, selected: boolean, isDisabled: boolean } */) {
  return <button type="button" value={props.value} disabled={isDisabled}>{props.children}</button>;
}
...
return (<div className="experienceDisplayer">
  <ExperienceViewButton value="map" ...>Map</ListViewButton>
  <ExperienceViewButton value="list" ...>List</ListViewButton>
</div>)
```

Reply:
🤯🤯🤯🤯 Thanks for that comment!